packages/shared/src/endpoints/v3/public.endpoints.ts

```diff
@@ -26,9 +26,13 @@ const Handshake = Endpoint({
 
 const VideoRecommendations = Endpoint({
   Method: 'GET',
-  getPath: ({ videoId }) => `/v3/video/${videoId}/recommendations`,
+  getPath: ({ channelId, videoId }) =>
+    `/v3/channels/${channelId}/videos/${videoId}/recommendations`,
   Input: {
-    Params: t.type({ videoId: t.string }),
+    Params: t.type({
+      videoId: t.string,
+      channelId: t.union([t.string, t.undefined]),
```

Comment:
This should be just a `t.string` if we want to keep in the path params, otherwise I would change it to a `query` param